tgui/packages/tgui/interfaces/JobSelection.tsx

```diff
@@ -38,21 +38,22 @@ type Data = {
 export const JobEntry: SFC<{ job; department; act: Function }> = (data) => {
   const job = data.job;
   const department = data.department;
+  const jobIcon = job.icon || JobToIcon[job.name] || null;
   return (
     <Stack.Item fill>
       <Button
         width="100%"
```

Comment:
use fluid property

Reply:
Button doesn't have a fluid property, unless I'm missing something!

Reply:
It does!

Reply:
https://github.com/tgstation/tgstation/blob/master/tgui/packages/tgui/components/Button.js#L64
https://github.com/tgstation/tgstation/blob/master/tgui/packages/tgui/styles/components/Button.scss#L95-L99